lib/ecto/changeset.ex

```diff
@@ -719,6 +721,13 @@ defmodule Ecto.Changeset do
         {changeset, false}
       end
 
+    changeset =
+      if opts[:force_update_on_change] do
```

Comment:
`Keyword.get(opts, :force_update_on_change, true)` since we want to default to true.